ocw/lib/openstack.py

```diff
@@ -25,7 +25,7 @@ def client(self) -> None:
         if self.__client is None:
             self.__client = openstack.connect(
                 debug=bool(DEBUG),
-                insecure=True,  # Trust the certificate
+                insecure=False,
```

Comment:
I am pretty sure that `insecure=False` is default so you can just drop the line

Reply:
> I am pretty sure that `insecure=False` is default so you can just drop the line

Fixed.